deploy-web/src/components/newDeploymentWizard/CreateLease.tsx

```diff
@@ -269,7 +269,7 @@ export const CreateLease: React.FunctionComponent<Props> = ({ dseq }) => {
 
   return (
     <>
-      <NextSeo title="Create Deployment - Create Lease" />
+      <CustomNextSeo title="Create Deployment - Create Lease" url="https://deploy.cloudmos.io/new-deployment?step=create-leases" />
```

Comment:
Could we use `urlUtils.ts` for the canonical urls? Something like:
```
url={`https://deploy.cloudmos.io${UrlService.newDeployment({step:"create-leases"})}`}
```

Reply:
Yes you're right